Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+flake8 = "*"
+
+[requires]
```

Comment:
Did you run the command `pipenv --python 3` to create the `Pipfile`?
Because usually when running it we get `python_version = "3.9"` (or whichever python 3 version you have in your environment).